src/main/plugins/com.sophos.tocjs/integrator.xml

```diff
@@ -29,7 +32,7 @@ See the accompanying LICENSE file for applicable license.
             <isset property="output.file" />
         </not>
     </condition>
-    <property name="output.file" value="${output.dir}${file.separator}toctree.js"/>
+    <property name="output.file" value="${ditaot.render.dir}${file.separator}toctree.js"/>
```

Comment:
I liked `render` just because it's not the actual "output" as we've used the term -- it's not the file(s) returned to the user, rather it's the place we put everything we're rendering. But I really don't care too much so I'm happy to change.